frontend/shared/components/src/records/FinancialSupportSettingsView.vue

```diff
@@ -60,6 +60,19 @@
             <STInputBox title="Benaming">
                 <input v-model="priceName" class="input" :placeholder="FinancialSupportSettings.defaultPriceName">
             </STInputBox>
+
+            <hr>
+            <h2>Verhinder automatische toekenning</h2>
+            <p>Als een lid gebruik wil maken van financiële ondersteuning dan zal hij goekeuring moeten vragen van een beheerder. In afwachting daarvan kan het lid niet inschrijven. Als het lid een UiTPAS-nummer met kansentarief opgeeft dan krijgt het lid wel automatisch financiële ondersteuning.</p>
+            <Checkbox v-model="isPreventSelfAssignment">
+                Verhinder automatische toekenning van financiële ondersteuning door een lid
+            </Checkbox>
+
+            <template v-if="isPreventSelfAssignment">
+                <STInputBox title="Verduidelijking voor een lid als hij niet kan inschrijven" class="max extra-padding">
```

Comment:
```suggestion
                <STInputBox title="Verduidelijking voor een lid als hij niet kan inschrijven" class="max">
```

Ik zou de extra class voorlopig weglaten, denk dat dit generieker moet worden opgelost zodat we elke checkbox gewoon meer plaats geven als er tekst in staat. Alternatief is om STList te werken met checkbox erin, maar dat gebeurt nu ook nog niet altijd.

Kan je na het mergen nog oplossen, is geen blocker